src/backend/distributed/planner/multi_physical_planner.c

```diff
@@ -5271,7 +5271,7 @@ ActivePlacementList(List *placementList)
 
 		/* check if the worker node for this shard placement is active */
 		workerNode = FindWorkerNode(placement->nodeName, placement->nodePort);
-		if (workerNode != NULL)
+		if (workerNode != NULL && workerNode->isActive)
```

Comment:
Should there be a check for all placements being inactive?

Reply:
done